db/migrate/20160418232042_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.string :body
+      t.string :author
+      t.string :title
+      t.integer :post_id
```

Comment:
This is totally fine, and the way we showed everyone... but there is a slightly better way:

```
t.references :post
```

That still creates an integer post_id column, but it also adds an index and a foreign key constraint, so you can't create a comment that doesn't have a post_id that actually corresponds to a post in the posts table.